filter.js

```diff
@@ -61,7 +60,12 @@ class Filter {
 
 
     const filters = this.statusOptions.map(s => {
-      return `<li class='filter-item'><input class="filter" type="checkbox" id="filter-${s.name}" value="${s.name}" checked><span class="legend--item--swatch" style="background-color: ${s.accessibleColor}"></span><label for="filter-${s.name}">${s.label}</label></li>`
+      /** Open for receiving / distributing donations should have two colors */
+      if (s.accessibleColor === "#2c7bb6" || s.accessibleColor === "#abd9e9") {
+        return `<li class='filter-item'><input class="filter" type="checkbox" id="filter-${s.name}" value="${s.name}" checked><span class="legend--item--swatch" style="background-color: ${s.accessibleColor}"></span>/<span class="legend--item--swatch" style="background-color: #fdae61"></span><label for="filter-${s.name}">${s.label}</label></li>`
+      } else {
+        return `<li class='filter-item'><input class="filter" type="checkbox" id="filter-${s.name}" value="${s.name}" checked><span class="legend--item--swatch" style="background-color: ${s.accessibleColor}"></span><label for="filter-${s.name}">${s.label}</label></li>`
+      }
```

Comment:
There's a lot duplicated in both clauses here. Can we extract a variable and have the duplicated string at the end?